.github/workflows/mutation.yml

```diff
@@ -16,7 +16,6 @@ jobs:
         with:
           php-version: 8.0
           tools: composer:v2
-          extensions: pcov
```

Comment:
According to the shivammathur/setup-php@v2 docs the pcov extension is installed by default